gpytorch/lazy/constant_mul_lazy_tensor.py

```diff
@@ -113,19 +107,17 @@ def _size(self):
 
     def _t_matmul(self, rhs):
         res = self.base_lazy_tensor._t_matmul(rhs)
-        res_mat_shape = res.shape[len(self.base_lazy_tensor.batch_shape) :]
-        constant = self.constant.view(*self.constant.shape, *[1 for i in res_mat_shape])
-        res = res * constant
+        res = res * self.expanded_constant
         return res
 
     def _transpose_nonbatch(self):
-        return ConstantMulLazyTensor(self.base_lazy_tensor._transpose_nonbatch(), self.constant)
+        return ConstantMulLazyTensor(self.base_lazy_tensor._transpose_nonbatch(), self._constant)
 
     @property
-    def constant(self):
+    def expanded_constant(self):
         # Make sure that the constant can be expanded to the appropriate size
         try:
-            constant = self._constant.expand(self.base_lazy_tensor.batch_shape)
+            constant = self._constant.view(*self._constant.shape, 1, 1)
         except RuntimeError:
             raise RuntimeError(
                 "ConstantMulLazyTensor of size {} received an invalid constant of size {}.".format(
```

Comment:
Consistency nit: below you format as `... ({}) ...`, let's be consistent in that regard.